Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -743,9 +743,11 @@ inline void BuilderCustom::applyInitialFontFamily(BuilderState& builderState)
         if (CSSValueID sizeIdentifier = fontDescription.keywordSizeAsIdentifier())
             builderState.setFontSize(fontDescription, Style::fontSizeForKeyword(sizeIdentifier, false, builderState.document()));
     }
+
+    auto value = AtomString("initial"_s);
+    initialDesc.setOneFamily(value);
     if (!initialDesc.firstFamily().isEmpty())
         fontDescription.setFamilies(initialDesc.families());
```

Comment:
```suggestion
    fontDescription.setFamilies(initialDesc.families());
```

This is what I originally suggested. Doesn't this work?

Reply:
I doubt your fix is correct, since it will instead look up a font named "initial".

Reply:
I'm curious what crash you get, because the end result is actually the same, with the only difference that your version sets a bogus font-family before setting the correct one.